core/system/src/unvic.c

```diff
@@ -604,4 +561,19 @@ void unvic_init(void)
      * by IRQ number. For example, IRQ 0 has precedence over IRQ 1 if both have
      * the same priority level. */
     NVIC_SetPriorityGrouping(0);
+
+    /* Make sure the UNVIC_VECTOR_UNUSED is definitely NOT a valid box id. */
+    if (vmpu_is_box_id_valid(UNVIC_VECTOR_UNUSED)) {
```

Comment:
We wouldn't have to do this check if `UNVIC_VECTOR_UNUSED` were replaced with an invalid box ID define.